contracts/BC_fusion/TokenRecoverPortal.sol

```diff
@@ -93,6 +95,10 @@ contract TokenRecoverPortal is ReentrancyGuardUpgradeable, System {
     // This event is triggered whenever a call to #recover succeeds.
     event TokenRecoverRequested(bytes32 tokenSymbol, address account, uint256 amount);
 
+    function initialize() external initializer onlyCoinbase onlyZeroGasPrice {
+        approvalAddress = INIT_APPROVAL_ADDRESS;
```

Comment:
This `address public assetProtector = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa` need to change too.